website/package.json

```diff
@@ -2,7 +2,8 @@
   "scripts": {
     "examples": "docusaurus-examples",
     "start": "docusaurus-start",
-    "build": "docusaurus-build",
+    "dry-run": "node ../build/documentation/copy-package-readme.js --dry-run",
```

Comment:
I think I'd prefer to place new scripts inside our root `package.json`.  Everything inside `website` technically is the Docusaurus package.  So If we update ever, these scripts could get overwritten.  We will also look to remove Docusaurus down the road once we get our documentation to a place where we feel it's adequate for consumers and contributors.  We have a lot of people on-boarding, so hopefully this is sooner than later.